sources/osgViewer/input/source/InputSourceWebVR.js

```diff
@@ -62,57 +68,78 @@ utils.createPrototypeObject(
             if (!customEvent.worldScale) customEvent.worldScale = 1.0;
         },
 
-        _pollHeadset: function() {
+        _schedulePolling: function() {
+            if (this._pollInterval > 0) {
+                setInterval(this.pollHeadset.bind(this), this._pollInterval);
+            }
+        },
+
+        pollHeadset: function() {
+            if (!this._nbCallbacks) {
+                //don't poll if there is no callback registered.
+                return;
+            }
+
             if (!navigator.getVRDisplays) {
```

Comment:
could be done in the _schedulepolling once and for al ?

Reply:
Ok, I changed it.
Now we start polling whenever a callback is added and stop hen there are no more callbacks registered